### PR TITLE
Added automated test for ODBC FailOver functionality

### PR DESCRIPTION
**Pull request by bisenpiyush**
[##](url) Changes proposed in this pull request
Added automated test for ODBC Failover functionality

## Patch testing
NA
## ReleaseNotes.txt changes
NA

## Other PRs 
This PR linked to generated PR in snappy-store [https://github.com/SnappyDataInc/snappy-store/pull/480](url)

### CODE REVIEW

File: dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java [hunk 1]
```diff
@@ -86,6 +86,7 @@
   private static String simulateStreamScriptDestinationFolder = TestConfig.tab().stringAt(SnappyPrms.simulateStreamScriptDestinationFolder, dtests);
   public static boolean isLongRunningTest = TestConfig.tab().booleanAt(SnappyPrms.isLongRunningTest, false);  //default to false
   public static boolean isUserConfTest = TestConfig.tab().booleanAt(SnappyPrms.isUserConfTest, false);  //default to false
+  public static boolean isCppTest = TestConfig.tab().booleanAt(SnappyPrms.isCppTest, false);  //default to false
```
Comment (kneeraj) [inline dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: Why are there so many formatting changes in this file. Have you made change in this file?

I will also check. I think you should revert formatting changes in this file. Let QA folks handle that?

Comment (bisenpiyush) [reply dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: Reverted formatting changes

File: dtests/src/test/java/io/snappydata/hydra/rowTableCPP/rowTableCPPTest.conf [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+           io.snappydata.hydra.cluster.SnappyPrms-scriptArgs = ${timeduration},${scriptPWD}
+           threadGroups = snappyStoreThreads
+           maxThreads = 1
+           ;
+
+hydra.Prms-totalTaskTimeSec           = 900;
+hydra.Prms-maxResultWaitSec           = 1200;
+io.snappydata.hydra.cluster.SnappyPrms-isCppTest = true;
+/*
+
```
Comment (sonalsagarwal) [inline dtests/src/test/java/io/snappydata/hydra/rowTableCPP/rowTableCPPTest.conf]: Please add CLOSETASK/ENDTASK to stop the cluster. You need to add the existing conf for it:

INCLUDE $JTESTS/io/snappydata/hydra/northwind/stopEmbeddedModeCluster.conf;

Comment (bisenpiyush) [reply dtests/src/test/java/io/snappydata/hydra/rowTableCPP/rowTableCPPTest.conf]: Added

**Pull request merged**